docs/configuration.md

```diff
@@ -28,20 +49,26 @@ cull:
   concurrency: 1
 ```
 
+<a name="notebook-culler"></a>
 ### Notebook culler
-How long a server can be idle before culling itself is controlled by the `IDLE_TIMEOUT`
-environment variable. This is configured in `singleuser.extraEnv` in `values.yaml`.
+There are two settings which control how the notebooks cull themselves. The first is
+`c.NotebookApp.shutdown_no_activity_timeout` which specifies the period of inactivity
+(in seconds) before a server is shutdown. The second is `c.MappingKernelManager.cull_idle_timeout`
+which determines when kernels will be shutdown.
 
 ```yaml
-singleuser:
-  extraEnv:
-    IDLE_TIMEOUT: 14400 # 4 hours
+extraConfig:
```

Comment:
I don't think this section works for setting these values which are meant to go into the config for the user notebook image. The `extraConfig` is for the hub config.

So for our deployment, we write out a `jupyter_notebook_config.py` file with these settings and run that as part of the user notebook image build:

https://github.ibm.com/IBM-Q-Software/ibmq-jupyterhub-user-image/blob/a41f97353d5016a812a1726b200a72e0468f0241/Dockerfile#L56

Maybe for these docs just mention the config (which you did above) and then link to https://jupyter-notebook.readthedocs.io/en/stable/config_overview.html for how to setup config for the notebook server.